core/src/epicli/data/common/ansible/playbooks/roles/postgresql/molecule/debian-repmgr/verify.yml

```
@@ -0,0 +1,53 @@
+---
+- name: Verify
+  hosts: postgresql
+  gather_facts: false
+  tasks:
+    - name: Check that DB port is open
+      block:
+        - name: Ensure that DB port is open
+          wait_for:
+            host: 0.0.0.0
+            port: 5432
+            state: started
+            delay: 0
+            timeout: 3
+          register: port_status
+
+        - name: Verify that port status is not changed
+          assert:
+            that:
+              - not port_status.changed
+
+    - name: Check that PostgreSQL services are started and enabled
+      block:
+        - name: Ensure that postgresql service is started and enabled
+          systemd:
+            name: postgresql
+            enabled: true
```

```suggestion
            enabled: true
            state: started
```
Shouldn't we have also `state`?

However, I would use `service_facts` module instead of `systemd`.